cpp/src/cylon/CMakeLists.txt

```diff
@@ -27,6 +27,19 @@ else (CYLON_PARQUET)
     set(PARQUET_CYLON_FILES)
 endif (CYLON_PARQUET)
 
+if (CYLON_UCX)
+    set(UCX_CYLON_FILES
+            net/ucx/ucx_channel.hpp
+            net/ucx/ucx_channel.cpp
+            net/ucx/ucx_communicator.hpp
+            net/ucx/ucx_communicator.cpp
+            net/ucx/ucx_operations.hpp
+            net/ucx/ucx_operations.cpp
+            )
+else (CYLON_UCX)
+    set(UCX_CYLON_FILES)
```

Comment:
didnt understand this line

Reply:
This is to have the UCX implementation's files added to the project only if a flag is passed saying to build with UCX for Cylon. (Similar to what was done when adding Parquet)